.github/workflows/test_setup.yml

```diff
@@ -22,6 +22,6 @@ jobs:
       - name: Setup polaris
         uses: fairwindsops/polaris/.github/actions/setup-polaris@master
```

Comment:
I think build-ext job will fail until this PR is merged because the job checks out master branch which does not have these changes.